packages/server/lib/modes/run.ts

```diff
@@ -814,10 +814,6 @@ async function runSpecs (options: { config: Cfg, browser: Browser, sys: any, hea
     })),
   })
 
-  if (testingType === 'component') {
```

Comment:
E2E and CT should be closing the browser equivalently so this removal looks good to me!